administrator/components/com_templates/src/View/Template/HtmlView.php

```diff
@@ -258,7 +258,13 @@ protected function addToolbar()
 			// Add a copy template button
 			elseif ($this->type === 'home')
 			{
-				ToolbarHelper::modal('copyModal', 'icon-copy', 'COM_TEMPLATES_BUTTON_COPY_TEMPLATE');
+				$templateDetails = JPATH_SITE.DIRECTORY_SEPARATOR.'templates'.DIRECTORY_SEPARATOR.($this->template->name).DIRECTORY_SEPARATOR.'templateDetails.xml';
+				$templateXML = simplexml_load_file($templateDetails);
+
+				if ((string) $templateXML->namespace === null || (string) $templateXML->namespace === '')
```

Comment:
```suggestion
				if ($templateXML->namespace === null || (string) $templateXML->namespace === '')
```
Maybe it can be simplified even more, but anyway a `(string) $templateXML->namespace === null` seems not to make sense to me.